src/Website/Resources/deleteorder.php

```diff
@@ -0,0 +1,150 @@
+<?php
+// Initialize the session
+session_start();
+ 
+// Check if the user is logged in, if not then redirect him to login page
+if(!isset($_SESSION["loggedin"]) || $_SESSION["loggedin"] !== true){
+    header("location: login.php");
+    exit;
+}
+$message = " ";
+
+if($_SERVER["REQUEST_METHOD"] == "POST"){
+    require_once "config.php";
+    if ($link->connect_error) {
+    die("Connection failed: " . $conn->connect_error);
+    }
+    $date = date("mdY");
+    $email = filter_input(INPUT_POST, 'email');
+    $useraddress = filter_input(INPUT_POST, 'address');
+    $user = filter_input(INPUT_POST, 'user_id');
+    $ordernumber= filter_input(INPUT_POST, 'ordernumber');
+    $productquantity = filter_input(INPUT_POST, 'itemquant');
+    $orderdate = time();
+    $sql = "UPDATE sales_orders set status_ = 0 WHERE order_num like '%$ordernumber%' AND userID like '%$user%'"; 
```

Comment:
makes sense. we need to attempt to change this from boolean values to text so it is more readable to customers.